lib/src/components/NxTextLink/__tests__/NxTextLink.test.tsx

```diff
@@ -5,114 +5,106 @@
+    expect(renderEl()).not.toHaveClass('nx-text-link--external');
+    expect(renderEl({ external: true })).toHaveClass('nx-text-link--external');
   });
 
   it('adds the noreferrer rel if the noReferrer prop is true', function() {
-    expect(getShallow()).toHaveProp('rel', '');
-    expect(getShallow({ rel: 'foo' })).toHaveProp('rel', 'foo');
+    expect(renderEl()).toHaveAttribute('rel', '');
+    expect(renderEl({ rel: 'foo' })).toHaveAttribute('rel', 'foo');
 
-    expect(getShallow({ noReferrer: true })).toHaveProp('rel', 'noreferrer');
+    expect(renderEl({ noReferrer: true })).toHaveAttribute('rel', 'noreferrer');
 
-    expect(getShallow({ noReferrer: true, rel: 'foo' }).prop('rel')).toContain('noreferrer');
-    expect(getShallow({ noReferrer: true, rel: 'foo' }).prop('rel')).toContain('foo');
+    expect(renderEl({ noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('noreferrer');
+    expect(renderEl({ noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('foo');
   });
 
   it('adds the noreferrer rel if the external prop is true and noReferrer is not false', function() {
-    expect(getShallow({ external: true })).toHaveProp('rel', 'noreferrer');
+    expect(renderEl({ external: true })).toHaveAttribute('rel', 'noreferrer');
 
-    expect(getShallow({ external: true, rel: 'foo' }).prop('rel')).toContain('foo');
-    expect(getShallow({ external: true, rel: 'foo' }).prop('rel')).toContain('noreferrer');
+    expect(renderEl({ external: true, rel: 'foo' })?.getAttribute('rel')).toContain('foo');
+    expect(renderEl({ external: true, rel: 'foo' })?.getAttribute('rel')).toContain('noreferrer');
 
-    expect(getShallow({ external: true, noReferrer: true })).toHaveProp('rel', 'noreferrer');
-    expect(getShallow({ external: true, noReferrer: false })).toHaveProp('rel', '');
+    expect(renderEl({ external: true, noReferrer: true })).toHaveAttribute('rel', 'noreferrer');
+    expect(renderEl({ external: true, noReferrer: false })).toHaveAttribute('rel', '');
 
-    expect(getShallow({ external: true, noReferrer: true, rel: 'foo' }).prop('rel')).toContain('noreferrer');
-    expect(getShallow({ external: true, noReferrer: true, rel: 'foo' }).prop('rel')).toContain('foo');
+    expect(renderEl({ external: true, noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('noreferrer');
+    expect(renderEl({ external: true, noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('foo');
   });
 
   it('sets aria-disabled on the text link when the disabled prop is set', function() {
-    expect(getShallow()).toHaveProp('aria-disabled', false);
-    expect(getShallow({ disabled: true })).toHaveProp('aria-disabled', true);
+    expect(renderEl()).toHaveAttribute('aria-disabled', 'false');
+    expect(renderEl({ disabled: true })).toHaveAttribute('aria-disabled', 'true');
   });
 
   it('sets link role on the text link when the disabled prop is set', function() {
```

Comment:
The setting of the attribute itself isn't important.  What's important is what the actual computed role of the element is: it should always be `link` regardless of whether the link is disabled.  You can use `getByRole` to check this.

Reply:
> The setting of the attribute itself isn't important. What's important is what the actual computed role of the element is: it should always be `link` regardless of whether the link is disabled. You can use `getByRole` to check this.

Updated here: 03d46791ff22a5aff9a1b1c96a074e5d957adac8

Reply:
> Updated here: https://github.com/sonatype/sonatype-react-shared-components/commit/03d46791ff22a5aff9a1b1c96a074e5d957adac8

`toBe` doesn't work that way.  Use `.toBeInTheDocument()`.  (Make sure you put the `()` on in order to actually invoke the assertion).